pom.xml

```diff
@@ -169,6 +170,12 @@
             <version>${commons.io.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
```

Comment:
why?

Reply:
For @Nonnull, but I ended up importing the lombok @NonNull one through intellij autocompletion, Capital N.

https://github.com/skodjob/odh-e2e/pull/99/files#diff-5f8b29c20e2ca8808f99f1f138936a1c096352d68d90bdf433260fe726d26162R67

I changed it to what I intended in the first place, If lombok annotation is preferable, then ok with me, I can remove dependency.

This jsr dependency was already present in test code (transitively).